### PR TITLE
[12.0][ADD] website_sale_product_description

### PR DESCRIPTION
**Pull request by dalonsod**
This addon makes editable the HTML `website_description` field of `product.template` model in order to be shown within the e-commerce product page. See `README.rst` for more details

### CODE REVIEW

**Comment by dalonsod** [comment]
cc @chienandalu @eLBati @sergio-teruel

File: website_sale_product_description/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+With this addon you can edit the existing website HTML description for a 
+product. This description is shown within the e-Commerce product page.
```
Comment (sergio-teruel) [inline website_sale_product_description/readme/DESCRIPTION.rst]: Specify that this field  has been showed in frontend.

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
I have doubts about this, as the idea behind not showing this in the backend is because it uses the website builder for filling it, and so the loaded assets in the backend are not the same, and touching from backend can lead to unexpected results in frontend.

**Comment by dalonsod** [comment]
@pedrobaeza the goal of this addon is providing a product description exclusively used on e-commerce product page, maintained from backend, despite of the current sale description, alse used in other documents. 

I've found that apparently unused field (`website_description`) and I've decided to used it in this way. Maybe it should be replaced by a new `Html` field, e.g. `public_description`, I have no problem doing this change, and keep `website_description` without changes in its current behavior. What do you think?

**Comment by pedrobaeza** [comment]
Yes, if you want that, you mustn't use `website_description`, as it's already used for dragging and dropping website builder blocks from the frontend.

**Comment by dalonsod** [comment]
Thanks for the help, I'll rewrite the PR

**Comment by chienandalu** [comment]
I agree, an html field (`website_description_extra` or something like that) placed by the standard `website_description` field block in frontend and editable in backend could be more flexible

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Comment by dalonsod** [comment]
@OCA/e-commerce-maintainers is this PR ready to be merged? Thanks!

**Comment by dalonsod** [comment]
Sorry for insisting @OCA/e-commerce-maintainers can anybody check if this PR is ready to be merged? Thanks!

**Comment by gfcapalbo** [comment]
LGTM :+1:

**Pull request merged**